llap-server/src/java/org/apache/hadoop/hive/llap/io/api/impl/LlapIoImpl.java

```diff
@@ -308,9 +308,14 @@ public long evictEntity(LlapDaemonProtocolProtos.EvictEntityRequestProto protoRe
     LOG.debug("Starting proactive eviction.");
     long time = System.currentTimeMillis();
 
-    long markedBytes = dataCache.markBuffersForProactiveEviction(predicate, isInstantDeallocation);
-    markedBytes += fileMetadataCache.markBuffersForProactiveEviction(predicate, isInstantDeallocation);
-    markedBytes += serdeCache.markBuffersForProactiveEviction(predicate, isInstantDeallocation);
+    long markedBytes = 0;
+    if (useLowLevelCache) {
```

Comment:
`SimpleBufferManager` is used for `dataCache` when `useLowLevelCache` is `false`, so we should evict it as well